template/partial/impact_of_assault/impact_2.html

```diff
@@ -0,0 +1,21 @@
+<table class="ui large very padded definition table">
+    <tbody>
+      <tr>
+        <td class="three wide dark-orange">Physical/Physiological</td>
+      </tr>
+      <tr>
+		<td class="dark-green">
+	        <ul>
+	            <li>Physical injury from the assault itself</li>
+				<li>Pregnancy</li>
+				<li>Sexually Transmitted Infections</li>
+				<li>Self-medicating (i.e., alcohol/drug abuse)</li>
+				<li>Sleep disorders</li>
+				<li>Eating disorders</li>
+				<li>Physical triggers (sights, smells, sounds, touching) that may produce a physical or psychological reaction</li>
+				<li>Self-mutilation (self-inflicted cuts but not to commit suicide)</li>
+			</ul>
```

Comment:
This is a github issue. I have rechecked the indentation in all files. Its shown here. If you test the code in local you will see that indentation is proper. This happened to me in the previous PR as well. Attaching a screenshot for the same.
<img width="1280" alt="screen shot 2017-06-16 at 13 37 26" src="https://user-images.githubusercontent.com/13470643/27217816-1a4b215e-5299-11e7-87f9-2bfe8b0eea6b.png">